src/shipit_code_coverage/tests/test_github.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+
+import responses
+
+from shipit_code_coverage.github import GitHubUtils
+
+
+@responses.activate
+def test_get_commit(MERCURIAL_COMMIT):
+    responses.add(responses.GET, 'https://api.pub.build.mozilla.org/mapper/gecko-dev/rev/hg/{}'.format(MERCURIAL_COMMIT), body='{} foo bar'.format(MERCURIAL_COMMIT), status=200)  # noqa
```

Comment:
The response from this service is in the format `GIT_COMMIT MERCURIAL_COMMIT`. The fake response should be in the same format of a real one.

Also move it to the fixture.